libs/README.md

```diff
@@ -67,7 +67,7 @@ const audiusSdk = sdk({
 ### HTML + JS example
 
 ```js title="In web page"
-const audiusSdk = window.audiusSdk({ apiKey: 'Your API key goes here' })
+const audiusSdk = window.audiusSdk({ appName: 'Your appName goes here' })
```

Comment:
I think we had decided to prefer api key in the docs, but allow someone to specify app name as an alternative (in a callout)

Reply:
Updated! Now `apiSecret` is optional in `DevAppSchema`, so a user can pass only apiKey and omit apiSecret if they only want read functionality